lib/rules.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class Rules
```

Comment:
Nitpick: 'Rules' seems like a vague name for this class. When I saw it, I expected it to be a readme of how to play ttt. Maybe a better name for this would be "EndGameCondtions" or "GameOverLogic" since all the logic in this class is regarding when the game should end

Reply:
changed to WinRules to show that this is for the logic to win the game